### PR TITLE
Feature#10 front end

### PR DESCRIPTION
**Pull request by OchoaHub**
# Front End
 
## Description: Why did you change/implement that?
- Adding bootstrap, bootstrap-select-rails and bootstrap_form gem
- Adding styles for all pages in the app

## Testing: Help me how can I test or look at the changes?
1. Type `bundle install` in terminal for install gems of the project
2. Type `rails s` in terminal for initialize a rails server
- Go to **localhost:3000/admin** and login with _admin@example.com_ and _password_ to see the admin panel and explore it
- Go to **localhost:3000/sessions/login** and type any number in the input to see the messages the app shows

### CODE REVIEW

**Review by LDVera — APPROVED** [review]

File: app/controllers/checks_controller.rb [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
-      return 30
+    case month
+    when "2"
+      28
+    when "4", "6", "9", "11"
+      30
     else
-      return 31
+      31
     end
```
Comment (n0b0dy-su) [inline app/controllers/checks_controller.rb:36]: # nice

Is better to read this syntax of your code
:+1:

File: app/controllers/checks_controller.rb [hunk 2]
```diff
@@ -43,46 +45,78 @@ def get_employees()
...
-    query.map{|s| {status: s[0], hits: s[1].to_i, page_views: s[2].to_i} }
+    query.map { |s| { status: s[0], hits: s[1].to_i, page_views: s[2].to_i } }
     puts query.to_json
   end
 
-  def get_average(type_move)
-    check_ins =  get_check(type_move)
-    total_hours = get_total_hours(check_ins)
-    average = calculate_average(total_hours, check_ins.length)
+
```
Comment (MiguelOcegueraM) [inline app/controllers/checks_controller.rb:56]: nice syntax

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -49,4 +50,14 @@ def check_out
   def enough_for_today
     redirect_to sessions_login_path, :alert => "Checks for today completed"
   end
+
```
Comment (MiguelOcegueraM) [inline app/controllers/sessions_controller.rb:53]: this looks very fine!:)

**Review by MiguelOcegueraM — APPROVED** [review]
Everything seems fine, go ahead with the next step

**Review by LPMendoza — APPROVED** [review]
I think you have made a good job, I see your interfaces by me self wit you but I think it's important include images in your pull request, specially in this cases that is a interfaces change. Good Job guys!

**Review by jnegrete15 — APPROVED** [review]

**Pull request merged**